web/wp-content/themes/dctx2/template-parts/content-reliefagad.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-hj68by.php, but found phpcs-scan-hJ68by.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).